Documentation on filtering and sharing Kedro-Viz via stateful URLS. (#1860)
kedro-org/kedro-viz: docs/source/share_kedro_viz.md

3. **Publish and Share Kedro-Viz with Github Actions**: Use the `publish-kedro-viz` action available on the Github Marketplace to deploy Kedro-Viz for your Kedro project repository on GitHub Pages. For further information and usage instructions, please refer to [publish-kedro-viz](https://github.com/marketplace/actions/publish-kedro-viz)

## Filtering and sharing Kedro-Viz pipelines 

> Kedro-Viz allows user to focus on specific aspects of their data pipeline by filtering the visualization. Once filters are applied, the resulting visualization can be shared with others by sharing the URL

Reviewer: ```suggestion
Kedro-Viz allows you to focus on specific aspects of your data pipeline by filtering the visualization. Once filters are applied, you can share the resulting visualisation with others by copying the URL
```

?

Author: lol i had it as you before and then i changed the tone to 'users' :D

Reviewer: I usually follow https://ddbeck.com/second-person-is-ok/ and https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#style

> As filters are applied, the URL in the browser automatically updates to reflect the current state of your filtered pipeline. By selecting filters like type, tags, or focus mode through the user interface, users generate a URL that can be shared, granting others access to the same filtered view of the pipeline.

Reviewer: ```suggestion
As filters are applied, the URL in the browser automatically updates to reflect the current state of your filtered pipeline. By selecting filters like type, tags, or focus mode through the user interface, you can generate a URL that can be shared, granting others access to the same filtered view of the pipeline.
```
